functional-tests/support/action/addInput.js

```diff
@@ -1,11 +1,57 @@
 import setInputField from "@nice-digital/wdio-cucumber-steps/lib/support/action/setInputField";
+import clearInputField from "@nice-digital/wdio-cucumber-steps/lib/support/action/clearInputField";
 
 export const addEmail = () => {
 	setInputField("add", "test@test.co.uk", "//*[@name='email']");	
 };
-
 export const addPassword = () => {
     setInputField("add", "Password1", "//*[@name='password']");
 };
+export const addTitle = () => {
+    setInputField("add", "NSAIDs Automated test again", ".FormField__inner.field-size-full  [name=title]");
+};
+export const addSource = () => {
+    browser.click("[for=source] .Select-control .Select-placeholder");
+    browser.waitForVisible("#react-select-2--list", 5000);     
```

Comment:
Automatically generated selectors are fragile as they will break when the html changes. For example, if I was to add another dropdown before source, the source list selector would get updated and become #react-select-3--list, meaning the old select would break and the test will fail. A more robust selector would be [for='source'] .Select-menu. We can tell the Id #react-select-2--list is auto generated because it is generic meaning it would fit any list. If we had written the it by hand we will have given it a bespoke Id such as #source-list.